src/libs/OptionsListUtils.js

```diff
@@ -77,6 +78,43 @@ Onyx.connect({
     },
 });
 
+const policyExpenseReports = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT,
+    callback: (report, key) => {
+        if (!ReportUtils.isPolicyExpenseChat(report)) {
+            return;
+        }
+        policyExpenseReports[key] = report;
+    },
+});
```

Comment:
It seems like we are defining a very similar variable in [this PR](https://github.com/Expensify/App/pull/17077/files#diff-733528d957ae6a80eb863637fa5d54278c89c6121998f3574fa8c0aba935daebR57-R73), can we just use that instead (once that PR is merged)?

Reply:
nvm my comment above, just realized this is for `policyExpenseReports` and not `expenseReports`